models/request.js

```diff
@@ -0,0 +1,74 @@
+var User = require('./user');
+var Service = require('./service');
+module.exports = function(sequelize, DataTypes) {
+    var Request = sequelize.define("Request", {
+      recipient_id: {
+        type: DataTypes.INTEGER,
+        primaryKey:true,
+        allowNull: false,
+        autoIncrement: true,
+        unique: true
+      },
+      services_id: {
+        type: DataTypes.INTEGER,
+        primaryKey:true,
+        allowNull: false,
+        autoIncrement: true,
+        unique: true
+      },
+      finalized: {
+        type: DataTypes.BOOLEAN,
+        primaryKey:true,
+        allowNull: true,
+      },
+      detail: {
+        type: DataTypes.TEXT,
+        allowNull: false,
+        validate: {
+          len: [1]
+        }          
+      },
+      requester_rating: {
+        type: DataTypes.INTEGER,
+        primaryKey:true,
+        allowNull: true,
+        autoIncrement: true,
+        unique: true
+      },
+      bounty_points: {
+        type: DataTypes.INTEGER,
+        primaryKey:true,
+        allowNull: true,
+        autoIncrement: true,
+        unique: true
+      },
+      start_date: {
+          type: DataTypes.DATE,
+          allowNull: false
+      },
+      end_date: {
+          type: DataTypes.DATE,
+          allowNull: false
+      }
+    });
+    User.associate = function(models) {
+        // We're saying that a Post should belong to an Author
+        // A Post can't be created without an Author due to the foreign key constraint
+        Post.belongsTo(models.User, {
+          foreignKey: {
+            allowNull: false
+          }
+        });
+      };
+    Service.associate = function(models) {
+        // We're saying that a Post should belong to an Author
+        // A Post can't be created without an Author due to the foreign key constraint
+        Post.belongsTo(models.Service, {
+          foreignKey: {
+            allowNull: false
+          }
+        });
+      };
+    return Request;
```

Comment:
I'm realizing it will make more sense if we pluralize the model names since they're synonymous with table names.

So let's change `Request` to `Requests` here.

It makes more sense with the Sequelize method naming conventions like `Requests.findOne()` and `Requests.findAll()`